spotbugs-exclude.xml

```diff
@@ -6,4 +6,20 @@
             <Bug pattern="BC_UNCONFIRMED_CAST" />
         </And>
     </Match>
+    <Match>
+        <And>
+            <Class name="io.ballerina.stdlib.graphql.runtime.engine.Engine" />
+            <And>
+            <Method name="addListMembersFromVariableValue" />
+            </And>
+            <Bug pattern="BC_UNCONFIRMED_CAST" />
+        </And>
+    </Match>
+    <Match>
+        <And>
+            <Class name="io.ballerina.stdlib.graphql.runtime.engine.Engine" />
+            <Method name="addInputObjectFieldsFromVariableValue" />
+            <Bug pattern="BC_UNCONFIRMED_CAST" />
+        </And>
```

Comment:
We can combine these two by removing the method name.

Reply:
if we remove the method name, Isn't it excluding the unconfirmed cast in the whole 'Engine' class?

Reply:
That would be okay I think since we have similar castings all over this class. (This actually is a false positive, as we check the type tag before casting, but the spotbugs plugin expects us to use the `instanceof` check.)